.github/workflows/release.yml

```diff
@@ -7,8 +7,44 @@ on:
       - "v*"
 
 jobs:
+  srtool:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chain: ["fastchain", "parachain"]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Identify Rust toolchain version
+        id: rustc_version
+        run: |
+          RUSTC_VERSION=$(grep '^channel = ' rust-toolchain.toml | sed 's/channel = "\(.*\)"/\1/')
+          echo "value=$RUSTC_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Srtool build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.9.2
```

Comment:
Please use commit hash as a v instead of the release number for third party (non-github) actions